Dockerfile.development

```
@@ -1,4 +1,4 @@
-ARG FROM_IMAGE=dependabot/dependabot-core
+ARG FROM_IMAGE=ghcr.io/dependabot/dependabot-core
```

Yeah, as documented in https://github.com/dependabot/dependabot-core#building-images-from-scratch the new base image at the bottom of our docker stack is now `dependabot-updater-core`: 
https://github.com/dependabot/dependabot-core/blob/4c9072d8a5575c7b0d7cd29c32c6d4658687f9cc/Dockerfile.updater-core

```suggestion
ARG FROM_IMAGE=ghcr.io/dependabot/dependabot-updater-core
```